packages/edit-site/src/components/layout/style.scss

```diff
@@ -118,7 +118,7 @@
 		justify-content: flex-end;
 	}
 
-	& > div {
+	.edit-site-resizable-frame__inner {
```

Comment:
I'm not sure if we have guidelines on this but it seems like using the class can aid searches. This file already uses a few other classes that aren’t BEM elements of `edit-site-layout` so it’s not the only exception. I don’t know if that had to do with why `> div` was used.